### ISSUE TITLE
Warnings in DefaultAtlasConversionService

### ISSUE THREAD

**Issue by rhuss** [syndesisio/syndesis#1408]
When starting the syndesis-rest, then the following WARNINGs happen on DefaultAtlasConversionService. Looks like a doube initialization somewhere. Any ideas @igarashitm ?

```
2018-02-05 07:05:24.471  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.String aleady exists.
2018-02-05 07:05:24.471  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Boolean aleady exists.
2018-02-05 07:05:24.472  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Long aleady exists.
2018-02-05 07:05:24.472  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Character aleady exists.
2018-02-05 07:05:24.472  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Byte aleady exists.
2018-02-05 07:05:24.472  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Double aleady exists.
2018-02-05 07:05:24.473  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Float aleady exists.
2018-02-05 07:05:24.473  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Integer aleady exists.
2018-02-05 07:05:24.473  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Short aleady exists.
2018-02-05 07:05:24.477  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.String aleady exists.
2018-02-05 07:05:24.478  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Boolean aleady exists.
2018-02-05 07:05:24.478  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Long aleady exists.
2018-02-05 07:05:24.478  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Character aleady exists.
2018-02-05 07:05:24.479  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Byte aleady exists.
2018-02-05 07:05:24.479  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Double aleady exists.
2018-02-05 07:05:24.479  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Float aleady exists.
2018-02-05 07:05:24.479  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Integer aleady exists.
2018-02-05 07:05:24.479  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Short aleady exists.
2018-02-05 07:05:24.482  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.String aleady exists.
2018-02-05 07:05:24.482  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Boolean aleady exists.
2018-02-05 07:05:24.482  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Long aleady exists.
2018-02-05 07:05:24.483  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Character aleady exists.
2018-02-05 07:05:24.483  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Byte aleady exists.
2018-02-05 07:05:24.483  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Double aleady exists.
2018-02-05 07:05:24.483  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Float aleady exists.
2018-02-05 07:05:24.483  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Integer aleady exists.
2018-02-05 07:05:24.484  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Short aleady exists.
2018-02-05 07:05:24.485  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.String aleady exists.
2018-02-05 07:05:24.485  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Boolean aleady exists.
2018-02-05 07:05:24.485  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Long aleady exists.
2018-02-05 07:05:24.485  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Character aleady exists.
2018-02-05 07:05:24.485  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Byte aleady exists.
2018-02-05 07:05:24.485  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Double aleady exists.
2018-02-05 07:05:24.486  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Float aleady exists.
2018-02-05 07:05:24.486  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Integer aleady exists.
2018-02-05 07:05:24.486  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Short aleady exists.
2018-02-05 07:05:24.487  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.String aleady exists.
2018-02-05 07:05:24.487  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Boolean aleady exists.
2018-02-05 07:05:24.487  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Long aleady exists.
2018-02-05 07:05:24.487  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Character aleady exists.
2018-02-05 07:05:24.487  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Byte aleady exists.
2018-02-05 07:05:24.488  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Double aleady exists.
2018-02-05 07:05:24.488  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Float aleady exists.
2018-02-05 07:05:24.488  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Integer aleady exists.
2018-02-05 07:05:24.488  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Short aleady exists.
2018-02-05 07:05:24.489  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.String aleady exists.
2018-02-05 07:05:24.490  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Boolean aleady exists.
2018-02-05 07:05:24.490  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Long aleady exists.
2018-02-05 07:05:24.490  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Character aleady exists.
2018-02-05 07:05:24.490  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Byte aleady exists.
2018-02-05 07:05:24.490  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Double aleady exists.
2018-02-05 07:05:24.491  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Float aleady exists.
2018-02-05 07:05:24.491  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Integer aleady exists.
2018-02-05 07:05:24.491  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Short aleady exists.
2018-02-05 07:05:24.492  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.String aleady exists.
2018-02-05 07:05:24.492  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Boolean aleady exists.
2018-02-05 07:05:24.493  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Long aleady exists.
2018-02-05 07:05:24.494  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Character aleady exists.
2018-02-05 07:05:24.494  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Byte aleady exists.
2018-02-05 07:05:24.494  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Double aleady exists.
2018-02-05 07:05:24.494  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Float aleady exists.
2018-02-05 07:05:24.494  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Integer aleady exists.
2018-02-05 07:05:24.494  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Short aleady exists.
2018-02-05 07:05:24.495  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.String aleady exists.
2018-02-05 07:05:24.496  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Boolean aleady exists.
2018-02-05 07:05:24.496  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Long aleady exists.
2018-02-05 07:05:24.497  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Character aleady exists.
2018-02-05 07:05:24.497  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Byte aleady exists.
2018-02-05 07:05:24.497  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Double aleady exists.
2018-02-05 07:05:24.497  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Float aleady exists.
2018-02-05 07:05:24.497  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Integer aleady exists.
2018-02-05 07:05:24.498  WARN [-,,,] 1 --- [           main] i.a.core.DefaultAtlasConversionService   : Converter between java.lang.Object and java.lang.Short aleady exists.
2018-02-05 07:05:26.445  INFO [-,,,] 1 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: org.springframework.security.web.util.matcher.AnyRequestMatcher@1, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@5d3b58ca, org.springframework.security
```

**Comment by igarashitm** [comment]
Yep, this is something we need to fix, atlasmap/atlasmap#269

**Comment by gashcrumb** [comment]
Fixed upstream...
